code/base.py

```diff
@@ -22,6 +22,7 @@
 
 import g
 import buyable
+from buyable import cash, cpu, labor
```

Comment:
i suppose the intention of adding that was to use the imported vars directly, but it seems that never happened

Reply:
cpu is used, other not.
I added them all by simplicity.

Reply:
what i mean is when you add `from foo import bar` then you typically remove `import foo` and later in the code you replace `foo.bar` with just `bar`. and that obviously didnt happen here

Reply:
Because import buyable are used for other things as buyable.Buyable

Reply:
so i guess that addition could be removed then ?
`from buyable import cash, cpu, labor`
i dont see any change that would require this line.